docker-compose.yaml

```diff
@@ -76,27 +81,8 @@ services:
     restart: always
     stop_grace_period: 2m
   resotoshell:
-    # This container is not started by default. To start a shell use this command:
```

Comment:
I generally agree to remove this option -let's not remove it right away.
there might be people that still use it this way.

Reply:
This is going into `main` 3.0 though, so I say let's break it now.